pxr/usdImaging/plugin/usdShaders/shaders/shaderDefs.usda

```diff
@@ -124,6 +159,59 @@ def Shader "UsdPreviewSurface" (
         and a value of 1.0 means the surface point is completely unoccluded by 
         other parts of the surface. """
     )
+
+    color3f inputs:sheen = (0.0, 0.0, 0.0) (
+        doc = """This tints the specular highlight of the surface.  The sheen effect
+        is most noticeable on cloth-like materials.  Equivalent to sheen from
+        KHR_materials_sheen and sheen_color from Autodesk Standard Surface."""
+    ) 
+
+    float inputs:sheenRoughness = 0.0 (
+        doc = """Roughness of the sheen highlight.  This is a separate roughness control
+        for the sheen highlight, which is useful for cloth-like materials.  Equivalent to 
+        sheenRoughness from KHR_materials_sheen and sheen_roughness from Autodek Standard
+        Surface."""
+    )
+      
+    float inputs:transmission = 0.0 (
+        doc = """The base percentage of light that is transmitted through the surface.
+        Equivalent to transmission from KHR_materials_transmission and transmission
+        from Autodesk Standard Surface."""
+    )
+
+    color3f inputs:attenuationColor = (1.0, 1.0, 1.0) (
+        doc = """Attenuation color.  This is the color of light that is attenuated as it passes
+        through the surface.  This is useful for glass-like materials.  Equivalent to 
+        attenuationColor from KHR_materials_volume and transmission_color from Autodesk
+        Standard Surface."""
+    ) 
+
+    float inputs:attenuationDistance = +Inf (
+        doc = """Attenuation distance.  This is the distance that light travels through the
+        surface before it is attenuated.  This is useful for glass-like materials.  Equivalent to 
+        attenuationDistance from KHR_materials_volume and transmission_depth from Autodesk
+        Standard Surfaced."""
+    )
+
+    float inputs:iridescenceFactor = 0.0 (
+        doc = """The iridescence intensity factor."""
```

Comment:
Small nitpick, I'd avoid the terms `Factor` (and `Texture` if that's anywhere).  "Factor" is a glTF-specific term that is roughly the equivalent of a shader uniform, and its presence often coincides with a lack of texturemapping and/or is being used to enable an extension.

Here for example, one might call this simply `iridescence`, not `iridescenceFactor`.

Reply:
BTW I just can not use the term "specularColor" as it is already in USD Preview Surface with another meaning:

```
    color3f inputs:specularColor = (0.0, 0.0, 0.0) (
        doc = """Used only in the specular workflow.
             Specular color to be used.
             This is the color at 0 incidence. Edge color is assumed white.
             Transition between the two colors according to Schlick fresnel 
             approximation."""
    )
```

Given it defaults to (0,0,0) and it has a different meaning and may already be used in a lot of files, I feel I should introduce another term in seeking alignment with Autodesk and Khronos to avoid any potential confusing.